examples/Cookbook.ipynb

```diff
@@ -184,6 +184,47 @@
     "print(\"Is contract:\", address.is_smart_contract())"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Changing the default hrp\n",
+    "\n",
+    "We have a configuration class, called `LibraryConfig`, that only stores the **default hrp** of the addresses. The default value is `erd`. The hrp can be changed when instantiating an address, or it can be changed in the `LibraryConfig` class, and all the addresses created will have the newly set hrp.  "
```

Comment:
`that only stores ... ` - "for now" / "for the moment".

Maybe, in the future, we'll move the most of the constants (that we use for default values in various parts). That way, the constants will be "promoted" to static variables.